yb-voyager/src/srcdb/pg_dump_extract_schema.go

```diff
@@ -148,20 +148,20 @@ func parseSchemaFile(exportDir string) int {
 				alterAttachPartition.WriteString(stmts)
 			case "MATERIALIZED VIEW":
 				objSqlStmts["MVIEW"].WriteString(stmts)
-			case "COLLATION":
-				objSqlStmts["COLLATION"].WriteString(stmts)
 			default:
 				uncategorizedSqls.WriteString(stmts)
 			}
 		}
 	}
 
 	// merging TABLE ATTACH later with TABLE - to avoid alter add PK on partitioned tables
-	objSqlStmts["TABLE"].WriteString(alterAttachPartition.String())
+	if utils.ContainsString(exportObjectTypesList, "TABLE") {
```

Comment:
Not needed. The if condition on 164 will take care of this.

Reply:
Yep